test/functional/apps/timeline/_expression_typeahead.js

```diff
@@ -44,9 +44,8 @@ export default function ({ getPageObjects }) {
     it('should display function suggestions filtered by function name', async () => {
       await PageObjects.timeline.setExpression('.o');
       const suggestions = await PageObjects.timeline.getSuggestionItemsText();
-      expect(suggestions.length).to.eql(2);
+      expect(suggestions.length).to.eql(1);
```

Comment:
I think `.e` will be bringing up `.es` and `.elasticsearch` ?

Reply:
![Screen Shot 2021-04-14 at 11 48 53 AM](https://user-images.githubusercontent.com/12060704/114763255-75237a00-9d17-11eb-9bc0-0bd9289dca9c.png)

for me I only have the one expression. We changed .es and .elasticsearch to both be opensearch so for now it just returns the one.

Reply:
Got it that's what i meant, this is the reason of the change :)